src/infoview.ts

```diff
@@ -351,6 +360,27 @@ export class InfoProvider implements Disposable {
                     function selectFilter(value) {
                         vscode.postMessage({command: 'selectFilter', filterId: parseInt(value)});
                     }
+                    function setupHover() {
```

Comment:
Please move this function back to `infoview-ctrl.js`.

Reply:
I just tried this and I'm having some trouble with tslint. How do I get typescript to recognize `acquireVsCodeApi`?

Reply:
The quick fix is to remove `@ts-check` from that file.  Ideally we'd use the typescript compiler to compile that file as well (then we could use `import`, `declare function`, etc.) but I'm not sure how to set that up.